### ISSUE TITLE
Modify the encoder

### ISSUE THREAD

**Issue by yuyu19970716** [antifen/mitosis-nuclei-detection#6]
Hello, thank you so much for the work you do! I would like to ask, if I want to change the encoder to vit-large-p16, where should I change it? Now I use python ./tools/train.py configs/vision_transformer/vit-large-p16_pt-64xb64_in1k-224.py and download the pre-trained model weights, but there is the following problem, can you please tell me what is going on?

2025-06-11 15:43:40,461 - mmcls - INFO - workflow: [('train', 1)], max: 300 epochs
2025-06-11 15:43:40,461 - mmcls - INFO - Checkpoints will be saved to /home/ubuntu/LXY/FoCasNet/mmclassification/work_dirs/vit-large-p16_pt-64xb64_in1k-224 by HardDiskBackend.
2025-06-11 15:44:15,019 - mmcls - INFO - Epoch [1][100/282]     lr: 3.000e-05, eta: 8:06:35, time: 0.346, data_time: 0.022, memory: 18850, loss: 0.6551, grad_norm: 6.2098
2025-06-11 15:44:47,074 - mmcls - INFO - Epoch [1][200/282]     lr: 5.999e-05, eta: 7:48:27, time: 0.321, data_time: 0.000, memory: 18850, loss: 0.6240, grad_norm: 6.5496
2025-06-11 15:45:13,113 - mmcls - INFO - Saving checkpoint at 1 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 1200/1200, 328.7 task/s, elapsed: 4s, ETA:     0s2025-06-11 15:45:20,917 - mmcls - INFO - Epoch(val) [1][38]        f1_score: [ 0. 80.]
2025-06-11 15:45:55,064 - mmcls - INFO - Epoch [2][100/282]     lr: 1.146e-04, eta: 6:10:10, time: 0.341, data_time: 0.021, memory: 18850, loss: 0.6358, grad_norm: 7.5419
2025-06-11 15:46:27,129 - mmcls - INFO - Epoch [2][200/282]     lr: 1.446e-04, eta: 6:26:17, time: 0.321, data_time: 0.000, memory: 18850, loss: 0.6544, grad_norm: 7.4096
2025-06-11 15:46:53,173 - mmcls - INFO - Saving checkpoint at 2 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 1200/1200, 333.3 task/s, elapsed: 4s, ETA:     0s2025-06-11 15:47:00,830 - mmcls - INFO - Epoch(val) [2][38]        f1_score: [ 0. 80.]
2025-06-11 15:47:34,949 - mmcls - INFO - Epoch [3][100/282]     lr: 1.992e-04, eta: 5:51:40, time: 0.341, data_time: 0.021, memory: 18850, loss: 0.6566, grad_norm: 8.3719
2025-06-11 15:48:07,022 - mmcls - INFO - Epoch [3][200/282]     lr: 2.292e-04, eta: 6:03:56, time: 0.321, data_time: 0.000, memory: 18850, loss: 0.6791, grad_norm: 9.4642
2025-06-11 15:48:33,077 - mmcls - INFO - Saving checkpoint at 3 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 1200/1200, 336.0 task/s, elapsed: 4s, ETA:     0s2025-06-11 15:48:40,667 - mmcls - INFO - Epoch(val) [3][38]        f1_score: [35.36585366 83.33333333]
2025-06-11 15:49:14,815 - mmcls - INFO - Epoch [4][100/282]     lr: 2.837e-04, eta: 5:43:35, time: 0.341, data_time: 0.021, memory: 18850, loss: 0.6627, grad_norm: 8.4040
2025-06-11 15:49:46,906 - mmcls - INFO - Epoch [4][200/282]     lr: 3.137e-04, eta: 5:53:06, time: 0.321, data_time: 0.000, memory: 18850, loss: 0.6684, grad_norm: 8.9510
2025-06-11 15:50:12,994 - mmcls - INFO - Saving checkpoint at 4 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 1200/1200, 335.9 task/s, elapsed: 4s, ETA:     0s2025-06-11 15:50:20,923 - mmcls - INFO - Epoch(val) [4][38]        f1_score: [ 0. 80.]
2025-06-11 15:50:55,061 - mmcls - INFO - Epoch [5][100/282]     lr: 3.682e-04, eta: 5:38:43, time: 0.341, data_time: 0.021, memory: 18850, loss: 0.6922, grad_norm: 10.2896
2025-06-11 15:51:27,151 - mmcls - INFO - Epoch [5][200/282]     lr: 3.982e-04, eta: 5:46:23, time: 0.321, data_time: 0.000, memory: 18850, loss: 0.7152, grad_norm: 12.1501
2025-06-11 15:51:53,200 - mmcls - INFO - Saving checkpoint at 5 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 1200/1200, 335.4 task/s, elapsed: 4s, ETA:     0s2025-06-11 15:52:00,922 - mmcls - INFO - Epoch(val) [5][38]        f1_score: [50.  0.]
2025-06-11 15:52:35,063 - mmcls - INFO - Epoch [6][100/282]     lr: 4.526e-04, eta: 5:35:16, time: 0.341, data_time: 0.021, memory: 18850, loss: 0.6716, grad_norm: 9.6997
2025-06-11 15:53:07,127 - mmcls - INFO - Epoch [6][200/282]     lr: 4.826e-04, eta: 5:41:37, time: 0.321, data_time: 0.000, memory: 18850, loss: 0.6859, grad_norm: 11.1070
2025-06-11 15:53:33,174 - mmcls - INFO - Saving checkpoint at 6 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 1200/1200, 332.5 task/s, elapsed: 4s, ETA:     0s2025-06-11 15:53:40,794 - mmcls - INFO - Epoch(val) [6][38]        f1_score: [ 0. 80.]
